package.json

```diff
@@ -2,27 +2,59 @@
   "name": "testium-core",
   "version": "1.4.5",
   "description": "Juggling the bits and pieces to run integration tests",
+  "license": "BSD-3-Clause",
   "main": "lib/testium-core.js",
-  "scripts": {
-    "test": "eslint lib test && mocha",
-    "posttest": "npub verify"
-  },
-  "files": [
-    "lib"
-  ],
+  "homepage": "https://github.com/testiumjs/testium-core#readme",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/testiumjs/testium-core.git"
+    "url": "git+ssh://git@github.com/testiumjs/testium-core"
+  },
+  "bugs": {
+    "chat": "http://signup.testiumjs.com/",
+    "url": "https://github.com/testiumjs/testium-core/issues"
+  },
+  "scripts": {
+    "pretest": "eslint lib test",
+    "test": "mocha",
+    "posttest": "nlm verify"
+  },
+  "nlm": {
+    "license": {
+      "files": [
+        "lib"
+      ]
+    }
+  },
+  "dependencies": {
+    "bluebird": "^3.3.3",
+    "concat-stream": "^1.5.0",
+    "debug": "^2.2.0",
+    "find-open-port": "^1.0.0",
+    "lodash": "^4.6.1",
+    "mkdirp-then": "^1.1.0",
+    "qs": "^4.0.0",
+    "rc": "^1.1.1",
+    "read-package-json": "^2.0.0",
+    "selenium-download": "^2.0.0",
+    "subprocess": "~0.3.0",
+    "testium-cookie": "^1.0.0"
+  },
+  "devDependencies": {
+    "assertive": "^2.0.0",
+    "babel-core": "^5.8.25",
+    "babel-eslint": "^4.1.3",
+    "cookie": "~0.1.3",
+    "eslint": "^1.0.0",
+    "eslint-config-groupon": "^2.0.0",
+    "gofer": "^2.4.5",
+    "mocha": "^2.0.0",
+    "nlm": "^2.0.0",
+    "testium-driver-sync": "^1.0.3",
+    "testium-example-app": "^1.0.5"
   },
-  "keywords": [
-    "test",
-    "integration",
-    "selenium",
-    "webdriver"
-  ],
   "author": {
-    "name": "Sean Massa",
-    "email": "endangeredmassa@gmail.com"
+    "name": "Groupon",
```

Comment:
This field is used by our generator to set the git username & email for the automated publishing. Not setting it to Groupon will make every release commit have "Sean Massa" in the name field. Hope that's okay with you @EndangeredMassa..?